feature-group-definitions/constraint-validation.yml

```diff
@@ -1,3 +1,4 @@
+name: "Constraint Validation API"
```

Comment:
Includes "API" because there's also constraint validation that happens automatically, not triggered by an API.